packages/server/lib/routes.ts

```diff
@@ -233,6 +234,11 @@ app.use(staticSite);
 
 // -------
 // Error handling.
-app.use((e: any, req: Request, res: Response, _: any) => {
-    errorManager.handleGenericError(e, req, res, tracer);
+app.use((err: any, req: Request, res: Response<ApiError<'invalid_json'>>, _: any) => {
+    if (err instanceof SyntaxError && 'body' in err && 'type' in err && err.type === 'entity.parse.failed') {
```

Comment:
really brittle https://github.com/expressjs/express/issues/4065

Reply:
😢 
why did you added the `&& 'type' in err && err.type === 'entity.parse.failed'` to the condition compared to the solution in the express repo issue?

Reply:
SyntaxError is a generic error so anything can throw that (it's really a bad package), at least it can't be confused with anything else like this
https://stackoverflow.com/a/58136032